services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/structure/directory/AppConfigurationDirectory.java

```diff
@@ -104,7 +105,9 @@ private void addArchiveIfMessageValid(String archiveName, Message message, Confi
 
     ArchiveOnDisk appConfigurationFile = new ArchiveOnDisk(archiveName);
     appConfigurationFile.addWritable(new FileOnDisk("export.bin", message.toByteArray()));
-    countryDirectory.addWritableToAll(__ -> new AppConfigurationSigningDecorator(appConfigurationFile, cryptoProvider,
-        distributionServiceConfig));
+    countryDirectory.addWritableToAll(ignoredValue ->
+        new AppConfigurationSigningDecorator(appConfigurationFile, cryptoProvider,
+        distributionServiceConfig)
```

Comment:
can be moved to previous line

Reply:
There are two reasons why I think this improves readability and code quality:

First, `__` goes against Java naming conventions. A developer who first encounters this might ask themselves what the meaning of that variable name is or if it's even a special keyword with potential side effects.

Secondly, `ignoredValue` makes the purpose and usage of that variable explicit.

Reply:
I've now moved the line.